### DISCUSSION TITLE
Looking For Test Suite

### DISCUSSION THREAD

**Discussion by solokyo** [pgmoneta/pgmoneta#70] [General]
Hi, 

I made some refactoring modifications for future AES implementation (See #56 ). The build process looks fine but I want to make sure that these changes won't cause any bugs. It will be great if we have a test suite.

**Comment by jesperpedersen** [comment]
Unfortunately, there is no automated test suite at the moment.

For now, it is best to setup a local PostgreSQL instance that uses `password_encryption = scram-sha-256` as the default, and make that explicit in the `pg_hba.conf` file as well. You can use most of https://github.com/pgmoneta/pgmoneta/blob/main/doc/tutorial/01_install.md

It would be nice to have an automated test suite, but that is a topic for another thread.
